addons/delivery/models/delivery_carrier.py

```diff
@@ -56,14 +56,25 @@ class DeliveryCarrier(models.Model):
     zip_from = fields.Char('Zip From')
     zip_to = fields.Char('Zip To')
 
-    margin = fields.Integer(help='This percentage will be added to the shipping price.')
+    margin = fields.Float(help='This percentage will be added to the shipping price.')
     free_over = fields.Boolean('Free if order amount is above', help="If the order total amount (shipping excluded) is above or equal to this value, the customer benefits from a free shipping", default=False, oldname='free_if_more_than')
     amount = fields.Float(string='Amount', help="Amount of the order to benefit from a free shipping, expressed in the company currency")
 
+    can_generate_return = fields.Boolean(compute="_compute_can_generate_return")
+    return_label_on_delivery = fields.Boolean(string="Generate Return Label", required=True, default=False,
+    help="The return label is automatically generated at the delivery.")
```

Comment:
weird indent

Reply:
True Flase or Null?

Reply:
null will return false

Reply:
yes but there's a default value, and so it's more robust